code/game/gamemodes/cult/talisman.dm

```diff
@@ -1,255 +1,397 @@
+	invocation = "Ta'gh fara'qha fel d'amar det!"
+	health_cost = 5
+
+/obj/item/weapon/paper/talisman/emp/invoke(mob/living/user, successfuluse = 1)
+	. = ..()
+	user.visible_message("<span class='warning'>[user]'s hand flashes a bright blue!</span>", \
+						 "<span class='cultitalic'>You speak the words of the talisman, emitting an EMP blast.</span>")
+	empulse(src, 4, 8)
+
+
+//Rite of Disorientation: Stuns and inhibit speech on a single target for quite some time
+/obj/item/weapon/paper/talisman/stun
+	cultist_name = "Talisman of Stunning"
+	cultist_desc = "A talisman that will stun and inhibit speech on a single target. To use, attack target directly."
+	invocation = "Dream sign:Evil sealing talisman!"
+	health_cost = 10
+
+/obj/item/weapon/paper/talisman/stun/invoke(mob/living/user, successfuluse = 0)
+	if(successfuluse) //if we're forced to be successful(we normally aren't) then do the normal stuff
+		return ..()
+	if(iscultist(user))
+		to_chat(user, "<span class='warning'>To use this talisman, attack the target directly.</span>")
+	else
+		to_chat(user, "You see strange symbols on the paper. Are they supposed to mean something?")
+	return 0
+
+/obj/item/weapon/paper/talisman/stun/attack(mob/living/target, mob/living/user, successfuluse = 1)
+	if(iscultist(user))
+		invoke(user, 1)
+		user.visible_message("<span class='warning'>[user] holds up [src], which explodes in a flash of red light!</span>", \
+							 "<span class='cultitalic'>You stun [target] with the talisman!</span>")
+		var/obj/item/weapon/nullrod/N = locate() in target
+		if(N)
+			target.visible_message("<span class='warning'>[target]'s holy weapon absorbs the talisman's light!</span>", \
+								   "<span class='userdanger'>Your holy weapon absorbs the blinding light!</span>")
+		else
+			target.Weaken(10)
+			target.Stun(10)
+			target.flash_eyes(1,1)
+			if(issilicon(target))
+				var/mob/living/silicon/S = target
+				S.emp_act(1)
+			if(iscarbon(target))
+				var/mob/living/carbon/C = target
+				C.silent += 5
+				C.stuttering += 15
+				C.cultslurring += 15
+				C.Jitter(15)
+		user.drop_item()
+		qdel(src)
+		return
+	..()
+
+
+//Rite of Arming: Equips cultist armor on the user, where available
+/obj/item/weapon/paper/talisman/armor
+	cultist_name = "Talisman of Arming"
+	cultist_desc = "A talisman that will equip the invoker with cultist equipment if there is a slot to equip it to."
+	invocation = "N'ath reth sh'yro eth draggathnor!"
+
+/obj/item/weapon/paper/talisman/armor/invoke(mob/living/user, successfuluse = 1)
+	. = ..()
+	user.visible_message("<span class='warning'>Otherworldly armor suddenly appears on [user]!</span>", \
+						 "<span class='cultitalic'>You speak the words of the talisman, arming yourself!</span>")
+	user.equip_to_slot_or_del(new /obj/item/clothing/head/culthood/alt(user), slot_head)
+	user.equip_to_slot_or_del(new /obj/item/clothing/suit/cultrobes/alt(user), slot_wear_suit)

```

Comment:
these should probably use `equip_or_collect` so that they don't waste a talisman just because they're too stupid to take off their clothes first